### PR TITLE
Added NewReview, etc.

### PR DESCRIPTION
**Pull request by bencdelaney**
- Added the NewReview class
- Made model changes
- Added Play unit tests

### CODE REVIEW

File: statler/settings.py [hunk 1]
```diff
@@ -83,7 +83,7 @@
         'NAME': 'statler',
         'USER': 'postgres',
         # EPIPHANY: Let's reference an environment variable here. (Maybe for the username, too)
-        'PASSWORD': '12345', # It's super secure!
+        'PASSWORD': '12345',  # It's super secure!
```
Comment (bencdelaney) [inline statler/settings.py:86]: My IDE said two spaces before inline comments is a thing (pep?)

File: statler_api/api_models.py [hunk 2]
```diff
@@ -42,29 +45,42 @@ def __init__(self, dao=None, text=None, date=None):
...
-#
-#         # timestamp the new review right now.
-#         self.timestamp = time.time()
-#
-#     def toDao(self):
-#         dao = ReviewDAO()
-#
-#         dao.text = self.text
-#         dao.timestamp = self.timestamp
+        self.timestamp = dao.timestamp
```
Comment (bencdelaney) [inline statler_api/api_models.py:48]: NOTE: This is an API change. I should probably make the requisite spec and javascript changes.

File: statler_api/api_models.py [hunk 3]
```diff
@@ -42,29 +45,42 @@ def __init__(self, dao=None, text=None, date=None):
...
+
+        self.text = text
+        self.playUrlTitle = playUrlTitle
+
+        # TODO: Push timestamping into the database.
+        # timestamp the new review right now.
+        self.timestamp = datetime.now()
+
+        # Fetch the rating externally
+        self.rating = text_processing.rateReview(text)
```
Comment (bencdelaney) [inline statler_api/api_models.py:67]: Should this be here, or in the view?

File: statler_api/models.py [hunk 4]
```diff
@@ -14,14 +14,15 @@ class PlayDAO(models.Model):
     change over time will be store elsewhere."""
 
     # Note: these lengths are arbitrary.
-    url_title = models.CharField(max_length=32)
+    url_title = models.CharField(max_length=32, unique=True)
```
Comment (bencdelaney) [inline statler_api/models.py:17]: These are the primary look up value, so they ought to be unique, I think.

File: statler_api/api_models.py (hunk 3 above)
Comment (bencdelaney) [inline statler_api/api_models.py:67]: @chaneberg, do you have an opinion?

Comment (chaneberg) [inline statler_api/api_models.py:67]: Here is fine, that way no one can create an invalid review object

File: statler_api/models.py [hunk 5]
```diff
@@ -14,14 +14,15 @@ class PlayDAO(models.Model):
...
     title = models.CharField(max_length=256)
     director = models.CharField(max_length=256)
     # null refers to database structure. blank refers to Django validation.
     actors = models.CharField(max_length=256, blank=True)
     description = models.CharField(max_length=1024, blank=True)
-    photo = models.FilePathField(blank=True)
-    show_times = models.CharField(max_length=256, blank=True);
+    # TODO: Figure out file upload location
```
Comment (chaneberg) [inline statler_api/models.py:23]: Images will be at /static/image.jpg, I think

**Pull request merged**